src/components/MessageBox.vue

```diff
@@ -295,15 +295,15 @@
     });
 
 
-    watch(props.chat.messages, () => {
+    watch(() => props.chat.messages, () => {
         nextTick(() => {
             const myName = window.location.host.split('.')[0];
             const lastItem = props.chat.messages[props.chat.messages.length - 1].from;
-
-            if (myName !== lastItem) return;
+            
+            //if (myName !== lastItem) return;
             messageBoxLocal?.value?.scrollTo(0, messageBoxLocal.value.scrollHeight);
```

Comment:
This means if an user is scrolling through his chat ands some one sends a message, he will be scrolled fully down?